packages/atomic/src/components/search/result-template-components/atomic-quickview/atomic-quickview.tsx

```diff
@@ -37,6 +37,20 @@ export class AtomicQuickview implements InitializableComponent {
   @State()
   public quickviewState!: QuickviewState;
 
+  /**
+   * The sandbox attribute to apply to the quickview iframe.
+   *
+   * The quickview is loaded inside an iframe with a [sandbox](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe#attr-sandbox) attribute for security reasons.
+   *
+   * This attribute primarily exists in order to protect against potential XSS attacks that could originate from the document being displayed.
+   *
+   * By default, the sandbox attributes are: `allow-popups allow-top-navigation allow-same-origin`.
+   *
+   * `allow-same-origin` is not optional, and must always be included in the list of allowed capabilities in order for the component to function properly.
```

Comment:
Clear and well-written description Oli 👌 
I only have minor suggestions:

```suggestion
   * The `sandbox` attribute to apply to the quickview iframe.
   *
   * The quickview is loaded inside an iframe with a [`sandbox`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe#attr-sandbox) attribute for security reasons.
   *
   * This attribute exists primarily to protect against potential XSS attacks that could originate from the document being displayed.
   *
   * By default, the sandbox attributes are: `allow-popups allow-top-navigation allow-same-origin`.
   *
   * `allow-same-origin` is not optional, and must always be included in the list of allowed capabilities for the component to function properly.
```